test/credentials_test.dart

```diff
@@ -147,24 +144,87 @@ Future<void> main([List<String>? args]) async {
     // Make sure the email haven't been already registered in apllication.
 
     // Enter a valid email that is not registered
-    String username = "existing_email@mail.com";
-    String password = "SWV23R#@T#VFQDV";
-
+    const String _validUsername = "valid_email@mail.com";
     baasTest('1. Register a valid user for email confirmation', (configuration) async {
       final app = App(configuration);
       final authProvider = EmailPasswordAuthProvider(app);
-      await authProvider.registerUser(username, password);
+      await authProvider.registerUser(_validUsername, _strongPassword);
     }, appName: "emailConfirm", skip: "It is a manual test");
 
     baasTest('2. Take the recieved token from the email and confirm the user', (configuration) async {
-      //Enter valid token and tokenId from the received email
+      // Enter valid token and tokenId from the received email
       String token = "3a8bdfa28e147f38e531cf5aca93d452a11efc4fc9a81f00219b0cb29cfb93858f6b174123659a6ef47b58a2b80eac3b406d7803605c17ef44401ec6cf2c8fa6";
       String tokenId = "626934dcb4e7e5a0e2f1d85e";
 
       final app = App(configuration);
       final authProvider = EmailPasswordAuthProvider(app);
       await authProvider.confirmUser(token, tokenId);
-      final user = await retryLogin(3, app.logIn, Credentials.emailPassword(username, password));
+      final user = await retryLogin(3, app.logIn, Credentials.emailPassword(_validUsername, _strongPassword));
+      expect(user, isNotNull);
+    }, appName: "emailConfirm", skip: "Run this test manually after test 1 and after setting token and tokenId");
```

Comment:
I suggest we prefix all manual tests names with `Manual test`

Reply:
OK